src/redux/auth-reducer.tsx

```diff
@@ -44,19 +75,19 @@ const authReducer = (state = initialState, action) => {
   }
 };
 
-export const getAuthUserData = () => {
+export function getAuthUserData() {
   return async (dispatch) => {
     try {
       const response = await authAPI.me();
       let { id, email, login } = response.data;
-      dispatch(setAuthUserData(id, email, login, id ? true : false));
+      dispatch(setAuthUserData(id ? true : false, id, email, login));
```

Comment:
if you can compute isAuth from id, why not do in function itself and remove isAuth parameter